pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -99,7 +99,7 @@
     <RuntimeGroup Include="freebsd">
       <Parent>unix</Parent>
       <Architectures>x64</Architectures>
-      <Versions>11;11.0;11.1;11.2;11.3</Versions>
+      <Versions>11;12;13</Versions>
```

Comment:
I don't think we've ever removed anything.... is that safe to do, @ericstj?

Reply:
I imagine that would be breaking.  We need to look at what the RID calculation algorithm is doing on those releases to understand if its safe to omit the minor version.

Reply:
I was wondering the same. However never really had FreeBSD support so I thought it would be OK and easier to drag forward.

Reply:
@wfurt did you investigate what the host calculates as a RID on these releases? https://github.com/dotnet/core-setup/blob/b2bff948e10f7632a40799808f4d7c74b912df80/src/corehost/common/pal.unix.cpp#L445
how about platform abstractions? https://github.com/dotnet/core-setup/blob/master/src/managed/Microsoft.DotNet.PlatformAbstractions/RuntimeEnvironment.cs#L31-L36

Reply:
core-setup/get_current_os_rid_platform  will get freebsd.11 and freebsd.12 `PlatformApis.GetOSVersion` will do the same.

Reply:
Do you have any more thoughts on this @ericstj ?

Reply:
Sure, if you've confirmed through testing that minor version is omitted from these and they are never actually used than I'm fine removing them.